modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/item/selector/test/ContentDashboardFileExtensionItemSelectorViewTest.java

```diff
@@ -131,13 +131,20 @@ public void testGetData() throws Exception {
 	}
 
 	@Test
-	public void testGetDataWithNoFileEntries() throws Exception {
+	public void testGetDataWithoutNewFileEntries() throws Exception {
+		Map<String, Object> defaultData = _getData();
+
+		JSONArray defaultFileExtensionGroupsJSONArray =
+			(JSONArray)defaultData.get("fileExtensionGroups");
+
 		Map<String, Object> data = _getData();
```

Comment:
We are running getData twice right? This doesn't make a lot of sense :(, maybe we can delete this test.